controller/metrics/prommetrics.go

```diff
@@ -184,3 +187,17 @@ var (
 		[]string{"kind", "namespace", "name", "verb", "status_code"},
 	)
 )
+
+// MetricVersionGauge version info
+var (
+	MetricVersionGauge = prometheus.NewGaugeFunc(
+		prometheus.GaugeOpts{
+			Name:        "argo_rollout_info",
+			Help:        "Running Argo-rollouts version",
+			ConstLabels: prometheus.Labels{"argo_rollout_version": version.GetVersion().Version},
```

Comment:
Nits: 
1) Can we rename this `argo_rollouts_controller_info` ? Because we might have a metric endpoint for dashboard in the future and should be more specific about the component
2) Label can just be `version` (this is what prometheus does)

```
			Name:        "argo_rollouts_controller_info",
			Help:        "Running Argo-rollouts version",
			ConstLabels: prometheus.Labels{"version": version.GetVersion().Version},
```

Reply:
Updated as suggested. Please check